packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/recent.tsx

```diff
@@ -30,17 +25,16 @@ const RecentComponent = ({
     is_explanation_expand,
     recent_strategies,
     toggleExplanationExpand,
-    is_delete_modal_open,
     toggleStrategies,
 }: TRecentComponent) => {
     const [get_strategies, setStrategies] = React.useState([]);
 
     React.useEffect(() => {
         toggleStrategies(true);
-        getSavedWorkspaces().then((load_recent_strategies: []) => {
+        getSavedWorkspaces().then(load_recent_strategies => {
             setStrategies(load_recent_strategies);
         });
```

Comment:
```suggestion
        getSavedWorkspaces().then(recent_strategies => {
            setStrategies(recent_strategies);
        });
```

We are not loading anything here we are just fetching the recent strategies right?

Reply:
done ✅